examples/js/loaders/MMDLoader.js

```diff
@@ -2179,9 +2198,135 @@ THREE.MMDLoader.prototype.createMesh = function ( model, texturePath, onProgress
 			m.blendSrcAlpha = THREE.SrcAlphaFactor;
 			m.blendDstAlpha = THREE.DstAlphaFactor;
 
-			if ( p.envMap !== undefined ) {
+			if ( m.map !== null ) {
+
+				function checkTextureTransparency ( m ) {
+
+					m.map.readyCallbacks.push( function ( t ) {
+
+						// Is there any efficient ways?
+						function createImageData ( image ) {
+
+							var c = document.createElement( 'canvas' );
+							c.width = image.width;
+							c.height = image.height;
+
+							var ctx = c.getContext( '2d' );
+							ctx.drawImage( image, 0, 0 );
+
+							return ctx.getImageData( 0, 0, c.width, c.height );
+
+						};
+
+						function detectTextureTransparency ( image, uvs ) {
```

Comment:
I tried it but some models were displayed incorrectly.
Indeed, this model was one of them.

https://twitter.com/superhoge/status/674102177867022336

This detection seems necessary and some other MMD tools(i.e. MMD Plugin for Unity)
have similar detection logic.

(TBH, I didn't think this's necessary but I ran into models which are rendered incorrectly
 without the detection, so I decided to implement)